.circleci/config.yml

```diff
@@ -711,15 +711,31 @@ jobs:
           name: Install Python dependencies
           command: pip install -U tox nox pip
           no_output_timeout: 5m
-      - run:
-          name: Run tests
-          no_output_timeout: 10m
-          command: |
-            CI_PYTEST_PARALLEL=<< parameters.xdist >> \
-            tox run -v \
-              -e importer-system-<<parameters.tests>>-py<<parameters.python_version_major>><<parameters.python_version_minor>>  \
-              -- \
-              tests/pytest_tests/system_tests/test_importers/test_<<parameters.tests>>/
+      - when:
+          condition: { equal: [<<parameters.tests>>, wandb] }
+          steps:
+            - run:
+                name: Run tests
+                no_output_timeout: 10m
+                command: |
+                  CI_PYTEST_PARALLEL=<< parameters.xdist >> \
+                  tox run -v \
+                    -e importer-system-<<parameters.tests>>-py<<parameters.python_version_major>><<parameters.python_version_minor>>  \
+                    -- \
+                    tests/pytest_tests/system_tests/test_importers/test_<<parameters.tests>>/
+      - when:
+          condition: { equal: [<<parameters.tests>>, mlflow] }
+          steps:
+            - run:
+                name: Run tests
+                no_output_timeout: 10m
+                command: |
+                  CI_PYTEST_PARALLEL=<< parameters.xdist >> \
+                  tox run -v \
+                    -e importer-<<parameters.tests>>-py<<parameters.python_version_major>><<parameters.python_version_minor>>  \
+                    -- \
+                    tests/pytest_tests/system_tests/test_importers/test_<<parameters.tests>>/
```

Comment:
if you remove the system thing, you could just use a single step and won't need conditionals here...